src/pages/historico/Historico.js

```diff
@@ -0,0 +1,16 @@
+import { SimpleHeader } from 'components';
+import classes from './Historico.module.css';
+
+function Historico(props) {
+  return (
+    <>
+      <div className={classes.pageContainer}>
+        <SimpleHeader
+          title="Histórico de movimentações"                    
+        />
+      </div>
+    </>
```

Comment:
```suggestion
      <div className={classes.pageContainer}>
        <SimpleHeader
          title="Histórico de movimentações"                    
        />
      </div>
```
As tags div já estão agrupando o conteudo, dai não precisa dos <>.